src/main/java/graphql/execution/FieldValueInfo.java

```diff
@@ -36,10 +37,21 @@ public CompleteValueType getCompleteValueType() {
         return completeValueType;
     }
 
-    public CompletableFuture<ExecutionResult> getFieldValue() {
+    public CompletableFuture<Object> getValue() {
         return fieldValue;
     }
 
+    /**
+     * @return a promise to the value wrapped in an execution result
+     *
+     * @deprecated use {@link #getValue()} instead
+     */
+    @Deprecated
+    @DeprecatedAt(value = "2022-10-03")
+    public CompletableFuture<ExecutionResult> getFieldValue() {
+        return fieldValue.thenApply(value -> ExecutionResult.newExecutionResult().data(value).build());
+    }
```

Comment:
Deprecated just to find usages initially but really maybe we should deprecated it